src/instructlab/sdg/pipeline.py

```diff
@@ -148,3 +148,4 @@ def _parse_pipeline_config_file(pipeline_yaml):
 # This is part of the public API.
 SIMPLE_PIPELINES_PACKAGE = "instructlab.sdg.pipelines.simple"
 FULL_PIPELINES_PACKAGE = "instructlab.sdg.pipelines.full"
+CUSTOM_PIPELINES_DATA_DIR = "/usr/share/instructlab/sdg"
```

Comment:
Take a look at how instructlab has started using the XDG Base Directory spec - e.g. using the `platformdirs` library

I think we would want `iter_data_dirs()` - i.e. `$XDG_DATA_HOME:$XDG_DATA_DIRS` ? (something like that)

Reply:
I've updated it to search user and site data dirs.